lib/login_gov/hostdata/railtie.rb

```diff
@@ -0,0 +1,42 @@
+begin
+  require 'lograge'
+rescue LoadError => e
+  STDERR.puts 'lograge is required by the LoginGov::Hostdata::Railtie, please add it to the Gemfile'
+  raise e
+end
+
+if Gem::Version.new(Rails::VERSION::STRING) < Gem::Version.new('6.1')
+  raise 'LoginGov::Hostdata::Railtie needs rails >= 6.1 to log request info, please upgrade'
+end
```

Comment:
Question 3... should we make this a hard requirement? We have a small number of apps and this helps us keep things simple? But wanted to call it out as a weird thing, this this requirement forced me to upgrade dashboard just to use this new functionality: https://github.com/18F/identity-dashboard/pull/393

Reply:
I think it kind of aligns with the above. if we have rails requirements, the gem should be able to require rails and specify a version :/